reana_commons/openapi_specifications/reana_job_controller.json

```diff
@@ -181,8 +175,8 @@
                   "2e4bbc1d-db5e-4ee0-9701-6e2b1ba55c20": {
                     "cmd": "date",
                     "cvmfs_mounts": [
-                      "atlas-condb",
-                      "atlas"
+                      "atlas.cern.ch",
+                      "cms.cern.ch"
```

Comment:
A better example may be, to stay within the context of ATLAS, the `atlas.cern.ch` and `atlas-condb.cern.ch`.  The acces to the  condition database is often needed, the cross-ATLAS-CMS analyses are rare.